### PR TITLE
adding a markdown instruction for on call engineers

### PR DESCRIPTION
**Pull request by mwrock**
Signed-off-by: Matt Wrock <matt@mattwrock.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: ON_CALL.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+
+* SSH to the `builder-datastore` instance
+* `su` to the `hab` user: `sudo su - hab`
+* run `/hab/pkgs/core/postgresql/9.6.1/20170514001355/bin/psql postgres`
+
+Note that while postgres is running as a habitat service and `hab pkg exec` can also run `psql`. `hab pkg exec` will run `psql` with busybox bash loaded and some of the `psql` navigation does not work well.
+
+Each of the builder services occupy their own database. `\l` will list all databases and you can connect to one using `\c <database name>`.
+
+Most of the databases are sharded, so you cannot simply start querying tables until you set your `SEARCH_PATH` to a shard. This can be potentially challenging. You need to know which shard has the data you are inteersted in. You navigate to a shard using `SET SEARCH_PATH TO shard_<number>;`. `\dn` will list all shards and can be helpful in determining if your database is sharded at all. If it is not, you just need to navigate to `shard_0`. Once set, you can run queries and also use `\dt` to list tables.
```
Comment (raskchanky) [inline ON_CALL.md]: `inteersted` = `interested`

**Comment by raskchanky** [comment]
tenor-84409517

**Comment by raskchanky** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/238152876).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/238152876).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**